Dnn.CommunityForums/themes/community-default/templates/TopicsView.ascx

```diff
@@ -201,18 +206,24 @@
 						<tbody>
 
 							<tr class="dcf-table-body-row">
-								<td class="dcf-col dcf-col-icon">[FORUMICONCSS]</td>
+								<td class="dcf-col dcf-col-icon">[FORUM:FORUMICONCSS|<div class="dcf-forum-icon" style="height: 30px; margin-right: 10px;"><i class="fa {0} fa-2x"></i></div>
```

Comment:
@johnhenley Shouldn't we avoid adding new `style` attributes to the HTML we're adding, and put it into CSS instead? 🤔 CC: @Timo-Breumelhof